java/org/contikios/cooja/dialogs/SerialUI.java

```diff
@@ -284,7 +284,7 @@ protected static void appendToTextArea(JTextArea textArea, String text) {
   }
 
   private static String trim(String text) {
-    return (text != null) && (!(text = text.trim()).isEmpty()) ? text : null;
+    return (text != null) &&  !(text = text.trim()).isEmpty() ? text : null;
```

Comment:
Please avoid unnecessary double spaces. Same for changes in other places.

Reply:
Fixed.

Reply:
Thanks for the update!